ksql-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -111,22 +118,22 @@ private void recordErrorRate(double value) {
   private Sensor configureErrorRate(Metrics metrics) {
     Sensor sensor = createSensor(metrics, metricGroupName + "-error-rate");
     sensor.add(
-        metrics.metricName("error-rate", this.metricGroupName,
-                           "The number of messages which were consumed but not processed. "
-                           + "Messages may not be processed if, for instance, the message "
-                           + "contents could not be deserialized due to an incompatible schema. "
-                           + "Alternately, a consumed messages may not have been produced, hence "
-                           + "being effectively dropped. Such messages would also be counted "
-                           + "toward the error rate."),
+        metrics.metricName(ksqlServiceId + "error-rate", this.metricGroupName,
```

Comment:
These are all backward incompatible changes. We should add a section to the `Upgrade notes` to include a mention that the metric names will change in 5.1.

Reply:
I'll add this to `Upgrading.rst`.